documentation/feature-flags.md

```diff
@@ -0,0 +1,51 @@
+# Node Agent Feature Flags
+
+As New Relic develops new instrumentation features, we may release some of them under feature flags--disabling them unless the agent is started with an optional flag to enable them. We do this to avoid side-effects on instrumented code when these still-experimental features are in development.  
+
+Within the agent, feature flags are exposed as properties of the `featureFlags` object, with flagged features defined as properties of the `featureFlags.prerelease` dictionary, and released features defined as elements of the `featureFlags.released` array. 
+
+Any prerelease flags can be enabled or disabled in your agent config by adding a `feature_flags` property. For example, if a hypothetical `example_new_instrumentation` behavior was behind a feature flag, adding `{ feature_flag: { example_new_instrumentation: true } }` to your `newrelic.js` would enable it.
+
+## Current prerelease feature flags
+
+#### express5 
```

Comment:
maybe I'm missing something but I don't see any code that is gated by this feature flag.  I think it can be "released"

Reply:
It does look like we've never removed it from the flags--the only `express5` mentions I'm seeing are related to the flag itself and the smoke test.

Reply:
yea express5 still is not GA. my guess is this was done first and we never followed through with experimental instrumentation. you could try to trace it via git blame